autopush-common/src/db/dual/mod.rs

```diff
@@ -0,0 +1,597 @@
+//! Dual data store.
```

Comment:
Welp, this will need to be refactored 😆 
https://github.com/mozilla-services/autopush-rs/pull/715

Reply:
Hmmm I wonder if this found it's way back in when rebasing. Ack, I will take it out.

Reply:
And yet, in https://github.com/mozilla-services/autopush-rs/pull/715 it's missing...